pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -449,7 +449,7 @@ protected Blob makeBlob(long oid) throws SQLException {
       return null;
     }
 
-    return makeBlob(getLong(i));
+    return makeBlob(getColumnOID(i));
```

Comment:
Currently `makeBlob` expects an id of a large object (see pg_largeobject) rather than id of the type. So this change is invalid